Slurm job started at 15/06/2023_22:39:38

NNODES = 4
SIMUNODES = 2
WORKERNODES = 2
NPROC = 4
NWORKER = 60
........['tcp://10.7.0.88:32831', 'tcp://10.7.0.88:33064', 'tcp://10.7.0.88:33235', 'tcp://10.7.0.88:34262', 'tcp://10.7.0.88:34378', 'tcp://10.7.0.88:34386', 'tcp://10.7.0.88:34431', 'tcp://10.7.0.88:35896', 'tcp://10.7.0.88:36453', 'tcp://10.7.0.88:37133', 'tcp://10.7.0.88:37291', 'tcp://10.7.0.88:37636', 'tcp://10.7.0.88:37855', 'tcp://10.7.0.88:39180', 'tcp://10.7.0.88:39291', 'tcp://10.7.0.88:39336', 'tcp://10.7.0.88:40608', 'tcp://10.7.0.88:41499', 'tcp://10.7.0.88:41640', 'tcp://10.7.0.88:41759', 'tcp://10.7.0.88:41990', 'tcp://10.7.0.88:43196', 'tcp://10.7.0.88:43202', 'tcp://10.7.0.88:43207', 'tcp://10.7.0.88:44290', 'tcp://10.7.0.88:45061', 'tcp://10.7.0.88:45464', 'tcp://10.7.0.88:45709', 'tcp://10.7.0.88:45899', 'tcp://10.7.0.88:46604', 'tcp://10.7.0.92:33337', 'tcp://10.7.0.92:33398', 'tcp://10.7.0.92:33892', 'tcp://10.7.0.92:34596', 'tcp://10.7.0.92:34697', 'tcp://10.7.0.92:35301', 'tcp://10.7.0.92:35389', 'tcp://10.7.0.92:35603', 'tcp://10.7.0.92:35612', 'tcp://10.7.0.92:36013', 'tcp://10.7.0.92:36282', 'tcp://10.7.0.92:36362', 'tcp://10.7.0.92:36428', 'tcp://10.7.0.92:36504', 'tcp://10.7.0.92:37229', 'tcp://10.7.0.92:38762', 'tcp://10.7.0.92:39225', 'tcp://10.7.0.92:41109', 'tcp://10.7.0.92:41167', 'tcp://10.7.0.92:41220', 'tcp://10.7.0.92:41403', 'tcp://10.7.0.92:43347', 'tcp://10.7.0.92:43665', 'tcp://10.7.0.92:43924', 'tcp://10.7.0.92:43972', 'tcp://10.7.0.92:44059', 'tcp://10.7.0.92:44191', 'tcp://10.7.0.92:44429', 'tcp://10.7.0.92:44751', 'tcp://10.7.0.92:45729']
dask.array<concatenate, shape=(12, 16384, 8192), dtype=float64, chunksize=(1, 8192, 4096), chunktype=numpy.ndarray>
Generated contract {'global_t': [(0, 12, 1)]}
Contract has been signed
Original arrays deleted
EST_ANALYTICS_TIME:            56.51184
Done
SIMULATION_TIME:      59.679293137000009 (avg: 4.973274428083334)
PDI_DELAY:      17.891285398000015 (avg: 1.490940449833335)
SIM_WTHOUT_PDI:       41.788007738999994 (avg: 3.482333978249999)
distributed.batched - INFO - Batched Comm Closed <TCP (closed) Client->Scheduler local=tcp://10.7.0.95:50934 remote=tcp://10.7.0.87:8786>
Traceback (most recent call last):
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/batched.py", line 93, in _background_send
    nbytes = yield self.comm.write(
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-tornado-6.2-lemkpcoofzciypjtwijrfhbso5bboah2/lib/python3.10/site-packages/tornado/gen.py", line 769, in run
    value = future.result()
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/comm/tcp.py", line 248, in write
    raise CommClosedError()
distributed.comm.core.CommClosedError
distributed.batched - INFO - Batched Comm Closed <TCP (closed) Client->Scheduler local=tcp://10.7.0.93:54880 remote=tcp://10.7.0.87:8786>
Traceback (most recent call last):
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/batched.py", line 93, in _background_send
    nbytes = yield self.comm.write(
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-tornado-6.2-lemkpcoofzciypjtwijrfhbso5bboah2/lib/python3.10/site-packages/tornado/gen.py", line 769, in run
    value = future.result()
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/comm/tcp.py", line 248, in write
    raise CommClosedError()
distributed.comm.core.CommClosedError
distributed.batched - INFO - Batched Comm Closed <TCP (closed) Client->Scheduler local=tcp://10.7.0.93:54878 remote=tcp://10.7.0.87:8786>
Traceback (most recent call last):
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/batched.py", line 93, in _background_send
    nbytes = yield self.comm.write(
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-tornado-6.2-lemkpcoofzciypjtwijrfhbso5bboah2/lib/python3.10/site-packages/tornado/gen.py", line 769, in run
    value = future.result()
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/comm/tcp.py", line 248, in write
    raise CommClosedError()
distributed.comm.core.CommClosedError
distributed.batched - INFO - Batched Comm Closed <TCP (closed) Client->Scheduler local=tcp://10.7.0.95:50932 remote=tcp://10.7.0.87:8786>
Traceback (most recent call last):
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/batched.py", line 93, in _background_send
    nbytes = yield self.comm.write(
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-tornado-6.2-lemkpcoofzciypjtwijrfhbso5bboah2/lib/python3.10/site-packages/tornado/gen.py", line 769, in run
    value = future.result()
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/comm/tcp.py", line 248, in write
    raise CommClosedError()
distributed.comm.core.CommClosedError
SUCCESS
tornado.application - ERROR - Exception in callback <bound method Client._heartbeat of <Client: 'tcp://10.7.0.87:8786' processes=60 threads=2400, memory=9.38 TiB>>
Traceback (most recent call last):
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-tornado-6.2-lemkpcoofzciypjtwijrfhbso5bboah2/lib/python3.10/site-packages/tornado/ioloop.py", line 921, in _run
    val = self.callback()
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/client.py", line 1099, in _heartbeat
    self.scheduler_comm.send({"op": "heartbeat-client"})
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/batched.py", line 136, in send
    raise CommClosedError(f"Comm {self.comm!r} already closed.")
distributed.comm.core.CommClosedError: Comm <TCP (closed) Client->Scheduler local=tcp://10.7.0.87:41476 remote=tcp://10.7.0.87:8786> already closed.

functions called:
HTML_ANALYTICS_TIME:      54.89
HTML_TRANSFER_TIME:       113.81
HTML_DESERIALIZE_TIME:    25.54
HTML_COMPUTE_TIME:        18.20
